x-pack/plugins/observability/server/types/rest_specs/slo.ts

```diff
@@ -43,6 +43,14 @@ const getSLOParamsSchema = t.type({
   }),
 });
 
+const findSLOParamsSchema = t.partial({
+  query: t.partial({
+    name_filter: t.string,
```

Comment:
nit: since you mentioned we will have extra filters later, maybe it would be better to not include `filter` in the name, so in the future with extra filters, we will not need to mention it for every filter (I see it is more clear this way but in general, seems a bit redundant, especially when having more filters)

Reply:
Good suggestion, I'll remove the filter part 👍🏻

Reply:
@maryam-saeidi Updated 👍🏻